app/models/authenticator.rb

```diff
@@ -16,7 +16,9 @@ def find_or_create_user
   end
 
   def update_user
-    user.update_attribute(:name, name) if user.name.blank?
+    if user.name.blank?
+      user.update(name: name)
+    end
```

Comment:
I'm fine with this, but we should probably change the Rubocop rules in another PR to be more lenient as well